x-pack/plugins/alerting/server/task_runner/task_runner.ts

```diff
@@ -601,7 +601,6 @@ export class TaskRunner<
     const scheduleDelay = runDate.getTime() - this.taskInstance.runAt.getTime();
 
     const event = createAlertEventLogRecordObject({
-      timestamp: runDateString,
```

Comment:
Is this field re-added later? [The event logger](https://github.com/elastic/kibana/blob/main/x-pack/plugins/event_log/server/event_logger.ts#L81) seems to only add a `@timestamp` field (and not a `timestamp` field)

Reply:
The `timestamp` used to get converted to `@timestamp` [here](https://github.com/elastic/kibana/blob/main/x-pack/plugins/alerting/server/lib/create_alert_event_log_record_object.ts#L51) within the `createAlertEventLogRecordObject` function. Now that it's no longer passed, it'll use the default event log's default properties.

Reply:
Awesome, thanks for clarifying!